src/helpers/table-sort-helper.js

```diff
@@ -8,6 +8,25 @@ function desc(a, b, orderBy) {
   return 0;
 }
 
+function statusDesc(a, b, competenceNumber) {
```

Comment:
We are sorting by competence status so the function should probably be named accordingly `competenceStatusDesc`.

I also think we should sort by the `validUntil` value instead of the status. Note that `validUntil` may be null, which means it is valid indefinitely, and should be considered as the highest possible value for sorting purposes.